README.md

```diff
@@ -217,134 +284,253 @@ spec:
 
 ```
 
+#### TensorBoard on GKE
 
-## Run the example
+On GKE you can store your event files on GCS and TensorBoard/TensorFlow
+can read/write directly to GCS.
 
-A simplistic TF program is in the directory tf_sample.
+```
+apiVersion: "mlkube.io/v1beta1"
+kind: "TfJob"
+metadata:
+  name: "tf-smoke-gpu"
+spec:
+  replica_specs:
+    - replicas: 1
+      tfPort: 2222
+      tfReplicaType: MASTER
+      template:
+        spec:
+          containers:
+            - image: gcr.io/tf-on-k8s-dogfood/tf_sample_gpu:latest
+              name: tensorflow
+              args:
+                - --log_dir=gs://my-bucket/logdir
+              resources:
+                limits:
+                  alpha.kubernetes.io/nvidia-gpu: 1
+          restartPolicy: OnFailure
+  tensorboard:
+    logDir: gs://my-bucket/logdir
 
-1. Start the example
+```
 
-    ```
-    helm install --name=tf-job ./examples/tf_job
-    ```
+#### Connecting to TensorBoard
 
-1. Check the job
+The TfJob operator will create a service named
+**tensorboard-$RUNTIME_ID** for your job. You can connect to it
+using the Kubernetes API Server porxy as follows
 
-    ```
-    kubectl get tfjobs -o yaml
-    ```
+Start the K8s proxy
+```
+kubectl proxy
+```
 
-## Project Status
+In a web-browser open up
 
-This is very much a prototype.
+```
+http://${PROXY}:8001/api/v1/proxy/namespaces/default/services/tensorboard-${RUNTIMEID}:80/
+```
 
-### Logging
+Depending on how you configure the service for TensorBoard and cluster
+you can make TensorBoard available without using the K8s proxy.
 
-Logging still needs work.
+## Staging code
 
```

Comment:
Should this be adressed in the README?
I think it falls outside the scope of this project, so maybe remove it from the README to get a clearer document.

Also while this alternatives are definitely doable, I am not sure they should really be encouraged, putting that in the README (especially if this falls under the tensorflow/ org) almost encourage people to do it this way.
What do you think?

Reply:
Good point. I removed it. 

I do think there's value in documenting how to setup K8s clusters to enable a team of datascientists/ml researchers to be successful. For that I do think access to a shared filesystem both locally and on the K8s cluster would be very valuable. I'll punt on that for now.